test/e2e/scenarios/reasons-for-appealing/reasonsForAppealing.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const paths = require('paths');
+
+Feature('Reason For Appealing');
+
+Before((I) => {
+    I.createTheSession();
+});
+
+After((I) => {
+    I.endTheSession();
+});
+
+Scenario ('Textfield allows to enter new lines', (I) => {
+
+    I.amOnPage(paths.reasonsForAppealing.reasonForAppealing);
```

Comment:
For consistency we always put the `I.amOnPage` directly after `I.createTheSession();`

Reply:
done